taurus/entity/tests/test_entity.py

```diff
@@ -12,3 +16,17 @@ def test_id_case_sensitivitiy():
     ingredient = IngredientRun(uids={'my_id': 'sample1'})
     assert ingredient.uids['my_id'] == 'sample1'
     assert ingredient.uids['MY_id'] == 'sample1'
+
+
+def test_content_hash():
```

Comment:
I'm not sure what the content_hash is to be used for, so I don't know what about it to test.